Resources/SteamLanguage/emsg.steamd

```diff
@@ -1691,11 +1695,16 @@ enum EMsg
 	ClientFSEnumerateFollowingListResponse = 7520;
 	FSGetPendingNotificationCount = 7521;
 	FSGetPendingNotificationCountResponse = 7522;
-	ClientFSOfflineMessageNotification = 7523;
-	ClientFSRequestOfflineMessageCount = 7524;
-	ClientFSGetFriendMessageHistory = 7525;
-	ClientFSGetFriendMessageHistoryResponse = 7526;
-	ClientFSGetFriendMessageHistoryForOfflineMessages = 7527;
+	ClientFSOfflineMessageNotification = 7523; removed "Renamed to ClientChatOfflineMessageNotification"
+	ClientFSRequestOfflineMessageCount = 7524; removed "Renamed to ClientChatRequestOfflineMessageCount"
+	ClientFSGetFriendMessageHistory = 7525; removed "Renamed to ClientChatGetFriendMessageHistory"
+	ClientFSGetFriendMessageHistoryResponse = 7526; removed "Renamed to ClientChatGetFriendMessageHistoryResponse"
+	ClientFSGetFriendMessageHistoryForOfflineMessages = 7527; removed "Renamed to ClientChatGetFriendMessageHistoryForOfflineMessages"

```

Comment:
Use `obsolete` instead of `removed` to continue generating the C# code for this enum value.